frontend/public/module/k8s/k8s-actions.ts

```diff
@@ -70,7 +70,11 @@ const actions = {
   getResources: () => dispatch => {
     dispatch({type: types.getResourcesInFlight});
     getResources_()
-      .then(resources => dispatch({type: types.resources, resources}))
+      .then(resources => {
+        // Cache the resources whenever discovery completes to improve console load times.
+        cacheResources(resources);
```

Comment:
Should this also be a Redux action?

Reply:
I guess this is where my Redux inexperience comes in. Is there a benefit to making it an action if we're not adding data to the store?